src/clients/host-client.js

```diff
@@ -339,7 +339,7 @@ class HostClient extends BaseEvernodeClient {
 
         // To obtain registration NFT Page Keylet and index.
         if (!tokenID)
-            tokenID = (await this.getRegistrationNft()).NFTokenID;
+            tokenID = (await this.getRegistrationUriTokens()).NFTokenID;
         const nftPageDataBuf = await EvernodeHelpers.getNFTPageAndLocation(tokenID, this.xrplAcc, this.xrplApi);
```

Comment:
Add a `TODO: Need to replace with URI token keylet` comment in places where `getNFTPageAndLocation` used for future reference